__tests__/utils/ethSigner.test.ts

```diff
@@ -321,4 +322,18 @@ describe('Ethereum signer', () => {
       );
     });
   });
+  describe('Ethereum address', () => {
```

Comment:
can we add a test for the `Invalid Ethereum Address Format` case as well?

Reply:
It's already the case (one byte missing).
I added a comment to clarify.